src/ocamlorg_toplevel/lib/dune

```diff
@@ -1,6 +1,13 @@
 (library
  (name ocamlorg_toplevel)
  (public_name ocamlorg.toplevel)
- (libraries lwt brr js_of_ocaml-tyxml)
+ (libraries
+  lwt
+  brr
+  js_of_ocaml-tyxml
+  rpclib
+  ppx_deriving_rpc
```

Comment:
We don't need the ppx as a library dependency. IIUC this confuses jsoo and bloats the final bundle by quite a lot. When built under the `release` profile I get (all uncompressed) for `stdlib-4.13.0.js`:

 - main: `220KB`
 - This PR: `2.3MB`
 - This PR with `ppx_deriving_rpc` removed from `libraries`: `243KB`
 
(I think I did this check correctly, would be good if someone could double-check)